### PR TITLE
Contrib clean up

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review

**Comment by Rick-Anderson** [comment]
If we keep this short we'll have a better chance of avoiding TL;DR

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,9 +66,9 @@ To highlight selected lines in a rendered snippet (usually renders as yellow bac
 
 ## Test your changes with DocFX
 
-Markdown editors can't render every feature of DFM. For example, they won't render snippets that reference separate code files.  To verify that your Markdown changes will render correctly when deployed to the docs.microsoft.com site, you can view them in your browser by using the [DocFX command line tool](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool) to create a locally hosted version of the site. DocFX requires the .NET Framework on Windows, or Mono for Linux or macOS.
+Test your changes with the [DocFX command line tool](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool), which creates a locally hosted version of the site.
```
Comment (tdykstra) [inline CONTRIBUTING.md]: This change loses the explanation that Wade suggested, and I agree, is needed at this point.

Comment (Rick-Anderson) [inline CONTRIBUTING.md]: It's still far too wordy. I don't agree you need it if you've come this far, but you should be able to roll it into the rewrite:

Markdown editors wont' render external code snippets, so you'll need to test your changes ...
